backend/voucher/views.py

```diff
@@ -195,17 +198,21 @@ class CreateVoucherList(generics.ListCreateAPIView):
 
     def get_queryset(self):
         queryset = Voucher.objects.all()
+
+        redeemer = self.request.query_params.get('nusnet_id', None) # retrieve Student id from GET
+        queryset = queryset.exclude(redeemer_id = redeemer)
+        # Do not include redeemed vouchers in the list
```

Comment:
Hi, which `list` is this referring to?

Reply:
the list of available vouchers returned from query

Reply:
If I recall correctly, the intended design is we'll still show the redeemed vouchers in the frontend but will simply just grey it out to the users. Is this correct? I kinda forgot. @radiankrisno @ThePrevailingOne

Reply:
Success criteria in Kanban only mentions redemption. If this is the case, can modify afterwards to suit that design.